packs/spells/animate-rope.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p>You cause a length or section of @UUID[Compendium.pf2e.equipment-srd.Item.Rope] or a rope-like object to animate and follow simple commands. You can give it two commands when you @UUID[Compendium.pf2e.actionspf2e.Item.Cast a Spell]{Cast the Spell}, and one command each time you @UUID[Compendium.pf2e.actionspf2e.Item.Sustain]{Sustain the Spell}.</p>\n<hr />\n<ul>\n<li><strong>Bind</strong> (attack) The Rope attempts to partially bind a creature. Attempt a @Check[spell-attack|defense:reflex]{spell attack roll} against the target's Reflex DC. If you succeed, the target takes a -10-foot circumstance penalty to its Speed (-20-foot on a critical success). This ends if the target @UUID[Compendium.pf2e.actionspf2e.Item.Escape]{Escapes} against your spell DC or breaks the Rope. (A standard adventuring Rope has Hardness 2, HP 8, and a Broken Threshold of 4.) @UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Animate Rope]</li>\n<li><strong>Coil</strong> The Rope forms a tidy, coiled stack.</li>\n<li><strong>Crawl</strong> The Rope inches along the ground like a snake, moving one of its ends 10 feet. The Rope must move along a surface, but that surface doesn't need to be horizontal.</li>\n<li><strong>Knot</strong> The Rope ties a sturdy knot in itself.</li>\n<li><strong>Loop</strong> The Rope forms a simple loop at one or both ends, or straightens itself back out.</li>\n<li><strong>Tie</strong> The Rope ties itself around a willing creature or an object that's unattended or attended by a willing creature.</li>\n<li><strong>Undo</strong> The Rope undoes one of its knots, ties, or bindings.</li>\n</ul>\n<hr />\n<p><strong>Heightened (+2)</strong> The range increases by 50 feet, and you can animate 50 more feet of Rope.</p>"
```

Comment:
This inline will be wrong if you make it with a different casting entry with a different attribute tied to it. Best to just add an attack roll to the spell itself if it doesn't have one.

Reply:
Oh, didn't think about that. I will do that.